packages/scss/src/components/inputs/textfield/_textfield.material.scss

```diff
@@ -40,7 +40,7 @@
 	.textfield-suffix {
 		position: absolute;
 		right: 0;
-		top: _component("textfield.material.padding-top") + _component("textfield.input.padding-vertical");
+		top: _component('textfield.material.suffix.top') + _component('textfield.input.padding-vertical');
```

Comment:
Same as for the compact mod. This doesn't look much more centered and adds variables

Reply:
✔️